regions/shapes/tests/test_polygon.py

```diff
@@ -5,18 +5,27 @@
 import astropy.units as u
 import numpy as np
 import pytest
-from astropy.coordinates import SkyCoord
+from astropy.coordinates import Latitude, Longitude, SkyCoord
 from astropy.tests.helper import assert_quantity_allclose
 from numpy.testing import assert_allclose, assert_equal
 
 from regions._utils.examples import make_example_dataset
 from regions._utils.optional_deps import HAS_MATPLOTLIB
 from regions.core import PixCoord, RegionBoundingBox, RegionMeta, RegionVisual
+from regions.shapes.circle import CircleSphericalSkyRegion
 from regions.shapes.polygon import (PolygonPixelRegion, PolygonSkyRegion,
+                                    PolygonSphericalSkyRegion,
                                     RegularPolygonPixelRegion)
 from regions.shapes.tests.test_common import (BaseTestPixelRegion,
-                                              BaseTestSkyRegion)
-from regions.tests.helpers import make_simple_wcs
+                                              BaseTestSkyRegion,
+                                              BaseTestSphericalSkyRegion)
+from regions.tests.helpers import assert_skycoord_allclose, make_simple_wcs
+
+
+def get_wcs():
```

Comment:
Is this used anywhere? I didn't see it at first glance. If it is used, would it make more sense to have it be a fixture?